content/en/news/release-notes.md

```diff
@@ -6,6 +6,31 @@ weight: 1
 
 For additional information check our [sprint demo videos](https://www.youtube.com/channel/UCcm2NzDN_UCZKk2yYmOpc5w) and [blogs](https://medium.com/kialiproject).
 
+## 1.55.0
+Sprint Release: August 12, 2022
+
+Features:
+
+* [(scalability) How to thin metrics to those required only by Kiali](https://github.com/kiali/kiali/issues/5151)
+* [Launch Kiali wizard scenarios from graph nodes](https://github.com/kiali/kiali/issues/4505)
+* [Milliseconds precision for sorting log entries in the Logs tab](https://github.com/kiali/kiali/issues/5246)
+* [Istio Workload Config Validation Optimization](https://github.com/kiali/kiali/issues/5153)
+* [(docs) document the use of kubectl operator in the kiali install docs](https://github.com/kiali/kiali/issues/5341)
```

Comment:
remove this - we didn't actually do anything for it... we just closed it... see the comment in that issue "After further review, never mind. ..."
```suggestion
```